js/qz-tray.js

```diff
@@ -909,6 +912,14 @@ var qz = (function() {
              * @memberof qz.websocket
              */
             getNetworkInfo: function(hostname, port, signature, signingTimestamp) {
```

Comment:
Two questions... 

1. Are you ok with the helper `version_2_0`
I puked a little in my mouth typing it but I don't know what else to call it.  Also, should we make this public?
2. I'm having difficulty calling this from the other namespace.  I'm trying: `_qz.websocket.getNetworkInfo` but it's not working.  I'd prefer not to use `qz` because that's exported and not guaranteed to be on the global scope for some implementations.

Reply:
1. `version_2_0` renamed to `isVersion(major, minor, patch)`, which we've needed for a long time so will be a welcome addition.  It's publicly available at `qz.api.isVersion(...)`
2. `_qz.websocket.getNetworkInfo` was moved to `_qz.tools.compatNetworking`.  @bberenz let me know if that's an OK place for this or not.